test/Microsoft.AspNetCore.Razor.Evolution.Test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/ComplexTagHelpers_DesignTime.mappings.txt

```diff
@@ -119,10 +119,15 @@ Source Location: (711:20,39 [23] TestFiles/IntegrationTests/CodeGenerationIntegr
 Generated Location: (5883:125,38 [23] )
 |DateTimeOffset.Now.Year|
 
-Source Location: (734:20,62 [7] TestFiles/IntegrationTests/CodeGenerationIntegrationTest/ComplexTagHelpers.cshtml)
-| - 1970|
-Generated Location: (5906:125,61 [7] )
-| - 1970|
+Source Location: (734:20,62 [2] TestFiles/IntegrationTests/CodeGenerationIntegrationTest/ComplexTagHelpers.cshtml)
+| -|
```

Comment:
Note: Now that we render line mappings for every token in a TagHelper attribute we may get unexpected perf consequences in the editor. Their biggest perf costs are due to projections so it'd be worth following up with them.